frontend/views/HomePage.jsx

```diff
@@ -13,6 +18,48 @@ function HomePage() {
       <div className="d-grid gap-2">
       <Button size='lg' onClick={() => {navigate("/login")}}>Prisijungti</Button>
       <Button size='lg' onClick={() => {navigate("/register")}}>Registruotis</Button>
+      {/* TODO: Remove everything below here, when we have backend */}
+      <Button 
+        size='sm' 
+        variant="warning" 
+        onClick={
+          () => {
+            const token = 0;
+            const roles = 2;
+            const email = "email@ktu.lt";
+            const password = "password";
+            setAuth({email, password, token, roles});
+            navigate('/admin/dashboard');
+          }
+        }>Prisijungti kaip Administratorius
+      </Button>
+      <Button 
+        size='sm' 
+        variant="warning" 
+        onClick={
+          () => {
+            const token = 0;
+            const roles = 0;
+            const email = "email@ktu.lt";
+            const password = "password";
+            setAuth({email, password, token, roles});
+            navigate('/');
+          }
+        }>Prisijungti kaip Darbuotojas</Button>
+      <Button 
+        size='sm' 
+        variant="warning" 
+        onClick={
+          () => {
+            const token = 0;
+            const roles = 1;
+            const email = "email@ktu.lt";
+            const password = "password";
+            setAuth({email, password, token, roles});
+            navigate('/');
+          }
+        }>Prisijungti kaip Darbdavys</Button>
+      {/* TODO: Remove everything above here, when we have backend */}
```

Comment:
This will have to be removed when we have the back end. All role assignment will be done in Login